src/data_collectors.js

```diff
@@ -80,22 +188,28 @@ function collectNodesData (doc, dialectData, ctx) {
 }
 
 /* Collects nodeMappings item scalar properties data. */
-function collectScalarPropsData (doc, node) {
+function collectScalarPropsData (doc, node, ontologyTerms) {
   const propsNodes = node.queryAll('shacl:property')
     .filter(prop => !!prop.query('shacl:datatype'))
   return propsNodes.map(prop => {
-    const propData = collectCommonPropData(doc, prop)
+    const propData = collectCommonPropData(doc, prop, ontologyTerms)
     propData.range = utils.parseHashValue(prop.query('shacl:datatype @id'))
     return propData
   })
 }
 
+/* Collects super-classes for a class */
+function collectSubClassOfLinks (doc, node) {
```

Comment:
Is this function implemented? It's description says it should "collect super-classes for a class" but query it performs doesn't use any arguments. Also `doc` is not used.